src/components/Navigation.stories.js

```diff
@@ -0,0 +1,12 @@
+import Navigation from './Navigation'
+
+export default {
+  title: 'Navigation',
+  component: Navigation,
+};
+
+const Template = args => <Navigation {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
+    
```

Comment:
Here you can reduce code if the component do not get any args...you can write: 
export const NavigationDefault = <Navigation />; 
instead of the line 8-11 ;)

Reply:
clean code, thx